src/System.Private.Uri/tests/FunctionalTests/IriRelativeFileResolutionTest.cs

```diff
@@ -81,6 +83,7 @@ public void IriRelativeResolution_CompareImplcitAndExplicitUncWithNoUnicode_AllP
         }
 
         [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)] // Unc paths must start with '\' on Unix
```

Comment:
Yes. fyi, the check is here: https://github.com/mono/mono/blob/master/mcs/class/referencesource/System/net/System/URI.cs#L3686